src/js/visual/configurator.js

```diff
@@ -912,6 +914,7 @@ ripe.Configurator.prototype._loadFrame = async function(view, position, options
         if (isReady) {
             await this._drawFrame(image, animate, duration);
         }
+        this.trigger("post_frame", view, position, false, options);

```

Comment:
Maybe move the `false` to the end so that we have (event name, operation args..., op/no-op).